src/formio/components/selectboxes.ts

```diff
@@ -0,0 +1,18 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type SelectboxesInputSchema = InputComponentSchema<string, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Concrete types
+ */
```

Comment:
can you add a `@todo` or implement the translation types for the values?

The equivalent Python types are here:

https://github.com/open-formulieren/open-forms/blob/7a788700c58d5cbbd7e8c791572051e12a698f66/src/openforms/formio/typing/base.py#L37